crates/core/src/rpc/accounts_data.rs

```diff
@@ -637,4 +666,106 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn test_get_block_commitment_basic() {
```

Comment:
This test isn't asserting anything - can we remove if the other tests are covering its cases?